website/docs/documentation/Pipelines/Smart-seq2_Single_Nucleus_Pipeline/README.md

```diff
@@ -0,0 +1,136 @@
+
+The pipeline has been scientifically validated by the BRAIN Institute. Read more in the [validation section](#validation).
+
+
+## Quick Start Table
+
+| Pipeline Features | Description | Source |
+|-------------------|---------------------------------------------------------------|-----------------------|
+| Assay Type | Smart-seq2 Single Nucleus | [Smart-seq2](https://www.nature.com/articles/nprot.2014.006)
+| Overall Workflow  | Quality control and transcriptome quantification | Code available from the [WARP repository](https://github.com/broadinstitute/warp/tree/develop/pipelines/skylab/smartseq2_single_nucleus/SmartSeq2SingleNucleus.wdl) in GitHub|
+| Workflow Language | WDL | [openWDL](https://github.com/openwdl/wdl) |
+| Genomic Reference Sequence (for validation)| GRCm38 mouse genome primary sequence | GENCODE GRCm38 [Mouse](http://ftp.ebi.ac.uk/pub/databases/gencode/Gencode_mouse/release_M23/GRCm38.primary_assembly.genome.fa.gz) |
+| Transcriptomic Reference Annotation (for validation) | Modified [M23](https://www.gencodegenes.org/mouse/release_M23.html) GTF built with the  [BuildIndices workflow](https://github.com/broadinstitute/warp/tree/master/pipelines/skylab/build_indices/BuildIndices.wdl)| GENCODE |
+| Aligner  | STAR (v.2.7.8a) | [STAR](https://github.com/alexdobin/STAR) |
+| QC Metrics | Picard (v.2.20.4) | [Broad Institute](https://broadinstitute.github.io/picard/)   |
+| Transcript Quantification | featureCounts (utilities for counting reads to genomic features) | [featureCounts(v2.0.2)](http://subread.sourceforge.net/)
+| Data Input File Format | File format in which sequencing data is provided | [FASTQ](https://academic.oup.com/nar/article/38/6/1767/3112533) |
+| Data Output File Formats | File formats in which Smart-seq2 output is provided | [BAM](http://samtools.github.io/hts-specs/), Loom (generated with [Loompy v.3.0.6)](http://loompy.org/), TXT (QC metrics and counts) |
+
+## Set-Up
+
+### Smart-seq2 Installation and Requirements
+
+The snSS2 workflow code can be downloaded by cloning the GitHub [WARP repository](https://github.com/broadinstitute/warp). For the latest release of snSS2, please see the [Smart-seq2 Single Nucleus changelog](https://github.com/broadinstitute/warp/tree/develop/pipelines/skylab/smartseq2_single_nucleus/SmartSeq2SingleNucleus.changelog.md).
+
+The workflow is deployed using [Cromwell](https://github.com/broadinstitute/cromwell), a GA4GH compliant, flexible workflow management system that supports multiple computing platforms.
+
+### Inputs
+
+There is an [example configuration (JSON) file](https://github.com/broadinstitute/warp/tree/develop/pipelines/skylab/smartseq2_single_nucleus/mouse_paired_example.json) available that you can to test the snSS2 workflow. It points to publicly available reference files and sample paired-end FASTQs.
```

Comment:
Just looking to confirm this input example is usable by someone not on our team - the files are all public?